app/styles/global.scss

```diff
@@ -43,6 +65,11 @@ $vz-sand: #eee6de;
 .crash-header-card-body {
   height: 400px;
   overflow-y: auto;
+  border-radius: 0px !important;
+}
+
+.mapboxgl-map {
+  border-radius: 0px;
```

Comment:
nixes the map's border radius, which was kinda awkward imo.

before:
<img width="484" alt="Screenshot 2025-01-28 at 3 27 29 PM" src="https://github.com/user-attachments/assets/7f7730c5-3d58-4d51-a6d9-5850b439c84d" />



after:
<img width="482" alt="Screenshot 2025-01-28 at 3 26 42 PM" src="https://github.com/user-attachments/assets/b35f81c7-2b84-41dc-bb7e-4cfd4db3b7c5" />